starknet-rpc-test/get_block_with_txs.rs

```diff
@@ -44,230 +39,151 @@ async fn fail_non_existing_block(#[future] madara: MadaraClient) -> Result<(), a
-            .unwrap(),
-            block_number: 1,
-            new_root: FieldElement::ZERO,
-            sequencer_address: FieldElement::from_hex_be(
-                "0x000000000000000000000000000000000000000000000000000000000000dead",
-            )
-            .unwrap(),
-            timestamp: block.timestamp,
-            transactions: vec![
-                StarknetTransaction::Invoke(InvokeTransaction::V1(InvokeTransactionV1 {
-                    transaction_hash: FieldElement::from_hex_be(
-                        "0x03be8055eece65051368768a6b92ae51e1a228edb04ebbd269e3bab555c4ed0e",
-                    )
-                    .unwrap(),
-                    max_fee: FieldElement::from_hex_be(MAX_FEE_OVERRIDE).unwrap(),
-                    signature: vec![
-                        FieldElement::from_hex_be("0x0676c246cb9d166ee69e20278767837e543a9982641d05e03ca3ea9bdb7629eb")
-                            .unwrap(),
-                        FieldElement::from_hex_be("0x066a8ee0282af011008df1a07bd30b20575b2a7b267a2ca5428eba7c8589b0ef")
-                            .unwrap(),
-                    ],
-                    nonce: FieldElement::ZERO,
-                    sender_address: FieldElement::TWO,
-                    calldata: vec![
-                        FieldElement::ONE,
-                        FieldElement::from_hex_be(FEE_TOKEN_ADDRESS).unwrap(),
-                        get_selector_from_name("transfer").unwrap(),
-                        FieldElement::ZERO,
-                        FieldElement::THREE,
-                        FieldElement::THREE,
-                        account_address,
-                        max_fee, // transfer uses the same max_fee as the deploy txn internally
-                        FieldElement::ZERO,
-                    ],
-                })),
-                StarknetTransaction::DeployAccount(DeployAccountTransaction {
-                    transaction_hash: FieldElement::from_hex_be(
-                        "0x02105f08ba02511ccef6ff6676a1481645ec33c9e0d9f7d654b0590aa6afb013",
-                    )
-                    .unwrap(),
-                    max_fee,
-                    signature: vec![
-                        FieldElement::from_hex_be("0x06bea565e0ac2450b1765ce3fec2ffd665f88b7c1c809a5713f795ab9641e133")
-                            .unwrap(),
-                        FieldElement::from_hex_be("0x00d8227bb300a313abb456689776dec594c2807b57824bf1159933e95946d227")
-                            .unwrap(),
-                    ],
-                    nonce: FieldElement::ZERO,
-                    contract_address_salt,
-                    constructor_calldata: vec![
-                        FieldElement::from_hex_be("0x0566d69d8c99f62bc71118399bab25c1f03719463eab8d6a444cd11ece131616")
-                            .unwrap(),
-                    ],
-                    class_hash,
-                }),
-            ],
-        },
+    assert_eq!(block.transactions.len(), 1);
```

Comment:
what was the issue with `assert_equal_blocks_with_txs`

Reply:
Most of what was check cannot be checked anymore.
hash, parent_hash, block_number, new_root
timestamp is fake anyway.

It leaves you with only status and sequencer_address.
sequencer_address is not related to this tests at all.

We can assert the block status, sure

Reply:
done